test/inductor/test_mkldnn_pattern_matcher.py

```diff
@@ -1808,12 +1808,12 @@ def matcher_check_fn():
                     mod,
                     (v,),
                     [
-                        "torch.ops.onednn.qlinear_pointwise.tensor",
-                        "torch.ops.onednn.qlinear_pointwise.binary",
+                        "aoti_torch_cpu__qlinear_pointwise_tensor",
+                        "aoti_torch_cpu__qlinear_pointwise_binary_tensor",
                     ],
                     [],
                     check_quantization=True,
-                    num_include_ops=[4, 4],
+                    num_include_ops=[2, 2],
```

Comment:
The number changes from 4 to 2 because:
Previously, for one occurrence of the functional call, the string `qlinear xxx` appeared twice:
```
PyTuple_SetItem(py_args_21, 0, PyUnicode_FromString("torch.ops.onednn.qlinear_pointwise.binary_tensor"));
...
throw std::runtime_error("PyObject_CallObject torch.ops.onednn.qlinear_pointwise.binary_tensor failed");
```

Now, it appeared only once:
```
AOTI_TORCH_ERROR_CODE_CHECK(aoti_torch_cpu__qlinear_pointwise_binary_tensor(convert_arrayref_tensor_to_tensor(buf8), x_scale_2, x_zp_2, _frozen_param18, convert_arrayref_tensor_to_tensor(buf13), convert_arrayref_tensor_to_tensor(buf14), &var_21, &var_22, 1.0, 0L, &var_23, 1.0, 0L, "sum", 1.0, "relu", var_array_3, 1, "", &buf16_handle));
```